<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_2">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>한컴 고딕</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>지역 선택</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>한컴 고딕</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>측정소 선택</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
